### PR TITLE
1023004

### PR DESCRIPTION
**Pull request by s1023004**
test

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1023004");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
Great. But there is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by s1023004** [comment]
yee~

**Comment by jrjang2** [comment]
Good. But there is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by s1023004** [comment]
finished

**Comment by s1023004** [comment]
finished

**Comment by jrjang2** [comment]
Good!

**Pull request merged**